src/config/cli_config.cpp

```diff
@@ -0,0 +1,155 @@
+#include "cli_config.hpp"
+
+#include <iostream>
+
+#include "cli_tools.hpp"
+
+using namespace std;
+
+namespace bpo = boost::program_options;
+
+namespace taraxa {
```

Comment:
This comment is not related just to this specific line + its more like opened question rather than requirement. For me this seems like separate "cli" stuff so I would personally put all files into the src/cli folder and also created new nemaspace taraxa::cli. Classes could be then called without Cli prefix + I would also put  devnet/tetsnet_config.hpp into the taraxa::cli namespace